index.js

```diff
@@ -8,12 +8,14 @@ module.exports = function (opts) {
   opts = _.defaults(opts || {}, {
     options: {},
     dest:    "cache",
-    src:     "assets"
+    src:     "assets",
+    ignore:  false
   });
```

Comment:
To keep this inline with all the other plugins. Use

```javascript
opts.ignore = babel._util.regexify(opts.ignore);
```

So then it can take a string etc.